src/pages/LectureVideo/components/LectureList/LectureList.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import ListItem from './components/ListItem';
+import styled from 'styled-components';
+
+function LectureList({
+  lectureList,
+  searchParams,
+  setSearchParams,
+  setVideoId,
+}) {
+  const goToLecture = () => {};
+
+  return (
+    <ListContainer onClick={goToLecture}>
+      {lectureList === undefined
+        ? null
+        : lectureList?.map(listItem => {
```

Comment:
```suggestion
      {lectureList &&
        lectureList?.map(listItem => {
```
없다면 안보여지도록 하는 로직이기에 && 연산자를 사용하는 것이 더 좋겠네요

Reply:
_완료_